### ISSUE TITLE
Allow for terser output

### ISSUE THREAD

**Issue by notorand-it** [opentofu/opentofu#756]
### OpenTofu Version

```shell
1.6.0-alpha2
```

### Use Cases

During complex executions (especially in pipelines) current verbosity can easily create thousands of lines that can force users to search through them.
There is no option to change output verbosity level.
Very terse output (ERROR mode) would just output errors.
Next level could output also warnings (WARNING mode).
Then you could add  "creating", "modifying" outputs (INFO mode).
Finally you would do what it does currently: a real lot of output (DEBUG mode).
The latter could be the default mode to keep "backwards compatibility".

### Attempted Solutions

A "decent workaround" I am currently using with terraform for the "INFO mode" looks like this:

```
terraform apply | { grep -F -v ': Still ' || true; }
``` 
It keeps the return value from `terraform` while filtering  out all those "Still ..." lines.

### Proposal

I think you need to tag output lines with a verbosity level to be later on filtered during actual output.

**Comment by davidjeddy** [comment]
👍  to providing proper logging levels. So many times I wished TF had more than `normal` and `debug` options. If we follow the Linux kernel levels we could have output verbosity based on the 8 levels:

KERN_EMERG (0)
The highest severity level warning. It is used when the system is unstable.

KERN_ALERT (1)
This log level is used when the error requires immediate user attention.

KERN_CRIT (2)
A log level of 2 is used to inform the user of critical software or hardware issues.

KERN_ERR (3)
Messages pertaining to a log level of 3 are usually used to inform the user of noncritical errors.

KERN_WARNING (4)
The default log level in many Linux distributions. It is used to display warnings about non-important errors.

KERN_NOTICE (5)
A log level of 5 is used to represent normal but significant conditions.

KERN_INFO (6)
KERN_INFO is used to display informational messages.

KERN_DEBUG (7)
KERN_DEBUG is used when the system is in debug mode. These messages contain information that is normally only useful when debugging a program.

**Comment by janosdebugs** [comment]
Hi folks, thank you for your input. We have discussed this issue in the core team and we were wondering if this issue may be closed in favor of #306? Thanks for your help!
